explainerdashboard/explainers.py

```diff
@@ -241,7 +242,9 @@ def __init__(
             col for col in self.regular_cols if not is_numeric_dtype(self.X[col])
         ]
         self.categorical_dict = {
-            col: sorted(self.X[col].unique().tolist()) for col in self.categorical_cols
+            col: sorted(
+                v for v in self.X[col].unique().tolist() if not pd.isna(v)
```

Comment:
not an expert on lightgbm, but wouldn't there be usecases where na would be a category? Or is that handled differently? How about by catboost or other libraries?